src/index.html

```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
                 <div class="modal-content">
                     <div class="modal-header">
                         <h5 class="modal-title" id="exampleModalLabel">Get Vowel Sounds, Encrypted</h5>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                            <span aria-hidden="true">&times;</span>
-                        </button>
                     </div>
                     <div class="modal-body">
                         The ciphertext of the record is available for download.
                         Once the contract reaches 10ETH, the decryption key will become available.
                     </div>
                     <div class="modal-footer">
-                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                        <button type="button" class="btn btn-primary">IPFS</button>
-                        <button type="button" class="btn btn-primary">Arweave</button>
-                        <button type="button" class="btn btn-primary">Bittorrent</button>
+                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
```

Comment:
Turns out we need `data-bs-dismiss` instead of `data-dismiss` with the current bootstrap version